lib/hammer_cli_katello/repository.rb

```diff
@@ -220,6 +221,8 @@ def execute
           File.open(file_path, 'rb') { |file| upload_file file }
```

Comment:
Ok, that makes sense. In that case, ACK - Thanks @jlsherrill! btw, this PR helped a bunch to test your Katello PR.